src/ray/gcs/tables.h

```diff
@@ -729,6 +729,29 @@ class ActorTable : public Log<ActorID, ActorTableData> {
     pubsub_channel_ = TablePubsub::ACTOR_PUBSUB;
     prefix_ = TablePrefix::ACTOR;
   }
+
+  /// Get all actor id synchronously.
+  std::vector<ActorID> GetAllActorID();
+
+  /// Get actor table data by actor id synchronously.
+  Status Get(const ActorID &actor_id, ActorTableData *actor_table_data);
+};
+
+/// Raw actor table.
```

Comment:
document that this table is only used for GCS-based actor management. And when we completely migrate to GCS service, we can remove the old actor table.